app/src/renderer/components/wallet/PageSend.vue

```diff
@@ -91,15 +91,15 @@ export default {
       this.sending = false
       this.$v.$reset()
     },
-    onSubmit () {
+    async onSubmit () {
```

Comment:
what does this have to do with bonding/unbonding?

Reply:
This was me improving some tests while fixing the tests. Should have maybe gone into another PR.